gradle.properties

```diff
@@ -0,0 +1,2 @@
+#path to Wowza home directory. Do not use qoutations marks.
+WMSINSTALL_HOME = C:\\Program Files (x86)\\Wowza Media Systems\\Wowza Streaming Engine 4.0.1
```

Comment:
Please use the linux path here as default and the windows and mac paths commented out.